recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.4.0rc1" %}
+{% set org = "cyclus" %}
+{% set branch = "develop" %}
```

Comment:
Though they should allow it for the tag. That's what I was trying to ask for here.